.github/workflows/ci.yml

```diff
@@ -534,7 +534,36 @@ jobs:
           - suite-5
         jdk:
           - 11
+          - 17
```

Comment:
would it be possible not to add it here and use inclusion instead of exclusion?

i don't want to accidentally run too many jobs if eg we add a new suite
and we hope to switch to j 17 eventually

Reply:
Inclusions are applied after exclusions so that some excluded combinations can be added back. But unfortunately, we can't do a conditional inclusion.

In #10895 we suggest adding a separate job that calculates the matrix, and we could add this kind of logic there more easily, so maybe we should postpone this a bit.

@wendigo asked to run 11 on PRs.

Reply:
> @wendigo asked to run 11 on PRs.

we still run 11 for other PT suites, so i though we can run 17 here.
anyway, whatever.

Reply:
I'm fine with either :)